docs/examples/server.pem

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
My apologies, I wasn't aware that GH would spam admins over this.  I naively based this example on some twisted docs that included keys and certs as part of the code, thinking that it would be a familiar presentation, but I realize that probably wasn't the best choice,

I will rework this to issue the requisite OpenSSL commands, etc.